pkg/lib/files/files.go

```diff
@@ -142,6 +142,10 @@ func WriteFile(data []byte, path string) error {
 	return nil
 }
 
+func IsAbs(path string) bool {
+	return strings.HasPrefix(path, "/") || strings.HasPrefix(path, "~/")
```

Comment:
It might be confusing if this returns true for tilde prefixed paths. Rename to IsAbsOrTildePrefixed